package.json

```diff
@@ -111,13 +111,13 @@
     "chokidar-cli": "^3.0.0",
     "eslint": "^8.57.0",
     "eslint-plugin-jsx-a11y": "^6.8.0",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.1.3",
```

Comment:
In order for `eslint-plugin-prettier` to work with `prettier` v3, we have to update it to v5. See https://github.com/prettier/eslint-plugin-prettier/blob/master/CHANGELOG.md#major-changes